src/test/java/org/sagebionetworks/bridge/exporter/scheduler/BridgeExporterSchedulerTest.java

```diff
@@ -0,0 +1,93 @@
+package org.sagebionetworks.bridge.exporter.scheduler;
+
+import static org.mockito.Matchers.eq;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static org.testng.Assert.assertEquals;
+
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.Table;
+import com.amazonaws.services.sqs.AmazonSQSClient;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.joda.time.DateTime;
+import org.joda.time.DateTimeUtils;
+import org.mockito.ArgumentCaptor;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+import org.testng.annotations.Test;
+
+public class BridgeExporterSchedulerTest {
+    private static final ObjectMapper JSON_OBJECT_MAPPER = new ObjectMapper();
+    private static final String TEST_SCHEDULER_NAME = "test-scheduler";
+    private static final String TEST_SQS_QUEUE_URL = "test-sqs-queue";
+    private static final String TEST_TIME_ZONE = "America/Los_Angeles";
```

Comment:
Since tests actually run on a computer set to seattle timezone, I like to setup for a *different* timezone so as to avoid unit test errors introduced by computer clock, though given that this is not going to be hosted or run by folks in another timezone, maybe not an issue?  (With iOS and devs across multiple timezones, it's a common mistake - "works on my machine!")